index.js

```diff
@@ -0,0 +1,67 @@
+import './module/dateSet.js';
+import BookApp, { sectionBooks } from './module/classUpdate.js';
+
+const inputTitle = document.querySelector('.title');
+const inputAuthor = document.querySelector('.author');
+const addBtn = document.querySelector('.add-btn');
+const footer = document.getElementById('footer');
+
+const books = new BookApp();
+
+const findHight = () => {
+  if (window.innerHeight > document.body.scrollHeight) {
+    footer.style.position = 'fixed';
+  } else {
+    footer.style.position = 'relative';
+  }
+};
+
+export default function loadBooks(bookCollection) {
```

Comment:
- [ ] I recommend you to use ES6 `arrow functions` instead of the `function `keywords. Because ES6 is today's trend, it is also about this milestone's theme.